improver_tests/synthetic_data/test_set_up_test_cubes.py

```diff
@@ -776,56 +786,56 @@ def test_error_unmatched_realizations(self):
                 self.data_2d, realizations=np.arange(realizations_len)
             )
 
-    def test_error_unmatched_height_levels(self):
-        """Test error is raised if the heights provided do not match the
+    def test_error_unmatched_vertical_levels(self):
+        """Test error is raised if the vertical levels provided do not match the
```

Comment:
The same long comment about duplicating this as I've given for `test_error_unmatched_height_levels` above.